lib/pages/common/multi_selection.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/pages/common/circular_checkbox.dart';
+
+class SelectionModel {
+  final String id;
+  final String label;
+  bool isChecked = false;
+
+  SelectionModel(this.id, this.label);
+
+  @override
+  String toString() {
+    return label;
+  }
+}
+
+class MultiSelection extends StatefulWidget {
+  final List<SelectionModel> items;
+  final ValueChanged<List<SelectionModel>> onChanged;
+
+  MultiSelection({required this.items, required this.onChanged});
+
+  @override
+  _MultiSelectionState createState() => _MultiSelectionState();
+}
+
+class _MultiSelectionState extends State<MultiSelection> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 65 * widget.items.length.toDouble(),
```

Comment:
Can we use `65.0` here, please? 🙏

Reply:
@markgravity Actually this is not a nice solution. I changed to use `shrinkWrap: true` in the next PR https://github.com/luongvo/flutter-survey/pull/65/files#diff-4076d830e0d70141795d852fb54827f1a78ab10ad25c4ccc09e0f2bf21894abcL31 👋